lhotse/cut.py

```diff
@@ -141,26 +150,10 @@ def truncate(
         )
 
     def overlay(self, other: AnyCut, offset_other_by: Seconds = 0.0, snr: Optional[Decibels] = None) -> 'MixedCut':
-        """
-        Overlay, or mix, this Cut with the `other` Cut. Optionally the `other` Cut may be shifted by `offset_other_by`
-        Seconds and scaled down (positive SNR) or scaled up (negative SNR).
-        Returns a MixedCut, which only keeps the information about the mix; actual mixing is performed
-        during the call to `load_features`.
-        """
-        assert self.num_features == other.num_features, "Cannot overlay cuts with different feature dimensions."
-        assert offset_other_by <= self.duration, f"Cannot overlay cut '{other.id}' with offset {offset_other_by}, " \
-                                                 f"which is greater than cuts {self.id} duration of {self.duration}"
-        new_tracks = (
-            [MixTrack(cut=other, offset=offset_other_by, snr=snr)]
-            if isinstance(other, Cut)
-            else other.tracks
-        )
-        return MixedCut(
-            id=str(uuid4()),
-            tracks=[MixTrack(cut=self)] + new_tracks
-        )
+        """Refer to mix() documentation."""
+        return mix(self, other, offset_right_by=offset_other_by, snr=snr)
```

Comment:
offset_right_by is a little confusing... I get that it refers to the right-most arg to the function but it looks like it might be something to do with the RHS (end) of the recording.  I wonder if offset_a_by and offset_b_by might be clearer than left/right?  Or cut1,cut2 instead of a,b?

Reply:
Alright, we'll need to rename it. Currently, it's not possible to specify the offset for the "left/a/1" track - to create the mix, we have to assume that one of the tracks is a reference point to which all the offsets refer. So I assume the left-most track in the mix will always have a zero offset.

We could rename the params like this: 

    def mix(reference_cut, mixed_in_cut, offset, snr): ...

would that make it clearer?